src/solver/simulation/adequacy_patch_runtime_data.cpp

```diff
@@ -0,0 +1,27 @@
+#include "adequacy_patch_runtime_data.h"
+
+bool AdequacyPatchRuntimeData::wasCSRTriggeredAtAreaHour(int area, int hour) const
+{
+    return csrTriggered[area].count(hour) > 0;
+}
+
+void AdequacyPatchRuntimeData::addCSRTriggeredAtAreaHour(int area, int hour)
+{
+    csrTriggered[area].insert(hour);
+}
+
+void AdequacyPatchRuntimeData::initialize(Antares::Data::Study& study)
```

Comment:
It's runtime data because it's initialized from runtime data (study->runtime).

Your remark was true before this patch (AdequacyPatchRuntimeData doesn't hold runtime data), but it becomes obsolete after new data member `csrTriggered` is added.